doc/admin/install/google_cloud.md

```diff
@@ -1,66 +1,101 @@
-# Deploy on Google Cloud
+# Deploy on Google Cloud Platform
 
-This tutorial shows you how to deploy `code-server` to a single node running on Google Cloud.
+This tutorial shows you how to deploy `code-server` to a single node running on
+Google Cloud Platform.
 
-If you're just starting out, we recommend [installing code-server locally](../../self-hosted/index.md). It takes only a few minutes and lets you try out all of the features.
+If you're just starting out, we recommend
+[installing code-server locally](self-hosted-docs). It takes only a few minutes
+and lets you try out all of the features locally.
 
----
-
-## Deploy to Google Cloud VM
-> Pre-requisite: Please [set up Google Cloud SDK](https://cloud.google.com/sdk/docs/) on your local machine
+If you get stuck or need help at anytime, [file an issue](create-issue),
+[tweet (@coderhq)](twitter-coderhq) or [email](email-coder).
 
-- [Open your Google Cloud console](https://console.cloud.google.com/compute/instances) to create a new VM instance and click **Create Instance**
-- Choose an appropriate machine type (we recommend 2 vCPU and 7.5 GB RAM, more depending on team size and number of repositories/languages enabled)
-- Choose Ubuntu 16.04 LTS as your boot disk
-- Expand the "Management, security, disks, networking, sole tenancy" section, go to the "Networking" tab, then under network tags add "code-server"
-- Create your VM, and **take note** of its public IP address.
-- Visit "VPC network" in the console and go to "Firewall rules". Create a new firewall rule called "http-8443". Under "Target tags" add "code-server", and under "Protocols and ports" tick "Specified protocols and ports" and "tcp". Beside "tcp", add "8443", then create the rule.
-- Copy the link to download the latest Linux binary from our [releases page](https://github.com/cdr/code-server/releases)
+[self-hosted-docs]: ../../self-hosted/index.md
+[create-issue]: https://github.com/cdr/code-server/issues/new?title=Improve+Google+Cloud+quickstart+guide
+[twitter-coderhq]: https://twitter.com/coderhq
+[email-coder]: mailto:support@coder.com?subject=Google%20Cloud%20quickstart%20guide
 
 ---
 
-## Final Steps
+## Deploy to Google Cloud VM
 
-- SSH into your Google Cloud VM
-```
-gcloud compute ssh --zone [region] [instance name]
-```
+> Pre-requisite: Please [set up Google Cloud SDK](gcloud-sdk) on your local
```

Comment:
This can be optional, you can go for a GUI-based guide route, or use Cloud Shell for gcloud commands.